Frameworks/UIKit/NSLayoutManager.mm

```diff
@@ -247,9 +247,9 @@ - (void)drawGlyphsForGlyphRange:(NSRange)range atPoint:(CGPoint)position {
         CTLineRef line = (CTLineRef)_ctLines[curLine];
         CGContextSaveGState(curCtx);
 
-        CGFloat ascent, leading;
+        CGFloat ascent, descent, leading;
         CTLineGetTypographicBounds(line, &ascent, nullptr, &leading);
```

Comment:
(i think ascent + leading would've been correct here in any case)

---
In reply to: [85786676](https://github.com/Microsoft/WinObjC/pull/1274#discussion_r85786676) [](ancestors = 85786676)